src/views/iframe.vue

```diff
@@ -40,19 +67,50 @@ export default {
     },
     fullScript() {
       // eslint-disable-next-line no-useless-escape
-      return `<script src="${process.env.VITE_BASE_URL}${this.scriptPath}"><\/script>`
+      return `<script src="${process.env.VITE_BASE_URL}${
+        this.scriptPath
+      }" ${this.options.join(" ")}><\/script>`
     },
   },
   mounted: function () {
```

Comment:
```suggestion
 watch: {
    options: function (): void {
      this.setIframeContainer()
    },
  },
  mounted: function () {
  this.setIframeContainer()
  },
 
```
Le linter seplaignait à cause de l'ordre déclaré des propriétés qui n'était pas respecté.